test/integration/targets/expect/tasks/main.yml

```diff
@@ -15,6 +15,10 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+- name: Install test requirements
+  pip:
+    name: pexpect
+    state: latest
```

Comment:
Only require a new enough version for the tests to pass, not the latest version. That way we can test whatever version is "native" to the environment running the test (when applicable), which gives better test coverage.

If you don't know what the minimum required version is, then don't require any specific version. We can fix the test later if we discover what the requirement is.